src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -216,13 +228,17 @@ trait WorkflowComponent {
         case None =>
```

Comment:
Actually I undid this. The compiler warns because the match isn't exhaustive if we do this.